Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM archlinux:base-devel AS build
```

Comment:
Have you investigated using the emscripten docker image?

Reply:
Not yet. Will look into it.